src/index.html

```diff
@@ -1,13 +1,62 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <section
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__photo">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="imac"
+        >
+      </div>
+
+      <div class="card__container">
+        <h3 class="card__title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h3>
+        <div class="card__code">
+          Код товара: 195434
+        </div>
+        <div class="card__rait">
+          <div class="card__eval">
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star card__star_turn-off"></div>
+          </div>
```

Comment:
```suggestion
          <div class="card__eval card__eval_4">
            <div class="card__star"></div>
            <div class="card__star"></div>
            <div class="card__star"></div>
            <div class="card__star"></div>
            <div class="card__star"></div>
          </div>
```
модификатор на блока в этом случаи